cds_dojson/schemas/records/books/book/document-v0.0.1.json

```diff
@@ -680,10 +704,10 @@
               "default": "author",
```

Comment:
default should have first uppercase, as changed in the enum below

Reply:
fixed.